### DISCUSSION TITLE
Selling 7 iSpindels

### DISCUSSION THREAD

**Discussion by heliocosta** [universam1/iSpindel#585] [General]
Hi everyone,
I live in London, U.K. and built 7 iSpindel a year ago or so that I'm non-longer using them.

I'm happy to sell them each for for £30 + shipping.

I'll give preference for those that live in the U.K. as it's easier to deliver and also who wants to buy them all.

![image](https://user-images.githubusercontent.com/1838945/175819187-c1bafa1d-b1d0-4e4d-ab65-f36ae46dede7.jpeg)

Send me an email helio.costa@gmail.com or WhatsApp +447488409498

Thanks
Helio

**Comment by thegreatgunbantoad** [comment]
Double check the temp sensors are reading OK on the first and last ones, those legs look a bit suspect.  I'd be inclined to stick them on eBay making sure that you properly reference this project and CherryPhillip and point out that you are in no way taking credit for the design or software.  Dependant on whether or not they have batteries you might not get £30 though.  Might also be worth making sure you update then to the latest SW.

Out of interest where did you get your tubes? I had to get mine from Lithuania!
